sql/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/test/IntegrationTestUtils.scala

```diff
@@ -90,6 +90,11 @@ object IntegrationTestUtils {
     Files.exists(Paths.get(filePath))
   }
 
+  lazy val isAssemblyJarsDirExists: Boolean = {
```

Comment:
Hmmm .. actually reading the code here, I think having a separate module might be better .. will it need a larger change?